dash/_callback.py

```diff
@@ -409,23 +409,14 @@ def add_context(*args, **kwargs):
 
                     job_fn = callback_manager.func_registry.get(long_key)
 
+                    ctx_value = AttributeDict(**context_value.get())
+                    ctx_value.ignore_register_page = True
+
                     job = callback_manager.call_job_fn(
                         cache_key,
                         job_fn,
                         func_args if func_args else func_kwargs,
-                        AttributeDict(
-                            args_grouping=callback_ctx.args_grouping,
-                            using_args_grouping=callback_ctx.using_args_grouping,
-                            outputs_grouping=callback_ctx.outputs_grouping,
-                            using_outputs_grouping=callback_ctx.using_outputs_grouping,
-                            inputs_list=callback_ctx.inputs_list,
-                            states_list=callback_ctx.states_list,
-                            outputs_list=callback_ctx.outputs_list,
-                            input_values=callback_ctx.input_values,
-                            state_values=callback_ctx.state_values,
-                            triggered_inputs=callback_ctx.triggered_inputs,
-                            ignore_register_page=True,
-                        ),
+                        ctx_value,
```

Comment:
When I try to run this with a background task, I get a callback error of `kombu.exceptions.EncodeError: Object of type Response is not JSON serializable`. It works if I pop `'dash_response'` and `'background_callback_manager'` from this dict though

Reply:
Do we have any automated tests for background tasks to help catch this...?